packages/design-system/src/components/UsaBanner/UsaBanner.tsx

```diff
@@ -29,123 +29,75 @@ export interface UsaBannerProps {
  */
 export const UsaBanner: React.FunctionComponent<UsaBannerProps> = (props: UsaBannerProps) => {
   const [isBannerOpen, setBannerOpen] = useState<boolean>(false);
-  const [shouldRenderMobileView, setShouldRenderMobileView] = useState<boolean>(false);
   const classes = classNames('ds-c-usa-banner', props.className);
   const id = props.id || uniqueId('gov-banner_');
 
-  useEffect(() => {
-    let media;
-    const onMediaChange = (evt) => {
-      setShouldRenderMobileView(evt.matches);
-    };
-
-    if (window) {
-      media = window.matchMedia('(max-width: 543px)');
-      media.addEventListener('change', onMediaChange);
-
-      setShouldRenderMobileView(media.matches);
-    }
-    return () => {
-      if (window) {
-        media.removeEventListener('change', onMediaChange);
-      }
-    };
-  }, []);
-
   const toggleBanner = () => {
     setBannerOpen(!isBannerOpen);
   };
 
-  const flagIcon = (
-    <UsaFlagIcon
-      className="ds-c-usa-banner__header-flag"
-      title={t('usaBanner.flagIconTitle')}
-      ariaHidden={false}
```

Comment:
USWDS sets this as `aria-hidden` - I think because this is a decorative-only icon. So setting `ariaHidden={false}` isn't needed here.